### PR TITLE
Feature/ttl

### PR DESCRIPTION
**Pull request by smousa**
Generic TTL manager for ephemeral data management or synchronization.

Note that changes are organized by commit.

### CODE REVIEW

File: commons/docker/ttl.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	"time"
+
+	"github.com/control-center/serviced/utils"
+	"github.com/zenoss/glog"
+)
+
+// DockerTTL is the ttl manager for stale docker containers.
+type DockerTTL struct{}
+
+// RunDockerTTL starts the ttl to reap stale docker containers.
```
Comment (wplu) [inline commons/docker/ttl.go]: RunTTL instead of RunDockerTTL

Comment (smousa) [inline commons/docker/ttl.go]: thanks, fixed

**Comment by tgjonestx** [comment]
\<nitPick\>new files should have Copyright 2015 instead of 2014\</nitPick\>

**Comment by smousa** [comment]
good catch, thanks Glenn

**Comment by wplu** [comment]
looks great!

**Pull request merged**